### DISCUSSION TITLE
File path select in a repeater

### DISCUSSION THREAD

**Discussion by analyserdmz** [directus/directus#2736] [Archive]
Hello,
is it possible to have a dialog where i can select some files from the server and keep their full-paths in a repeater field? My files are mp4 and are already uploaded to the server.

I can do it by manually adding their full paths to a repeating text field (JSON) but i am unable to find anything that would let me browse the server's file system. And adding them manually is a very time consuming process.

Thank you.

**Comment by benhaynes** [comment]
Is that just a "Multiple Files" interface?

If not, this would likely need to be a custom interface. You can duplicate whichever is closest to your requirements and adapt as needed. The relational repeater (coming in v9) should also work for this since it will relationally return the file info (including full URL) when one of the fields is a file.

**Reply by analyserdmz** [reply]
Yeah, i just need a "multiple files" interface which would allow me to point-and-click server-side video files, exactly!

**Reply by benhaynes** [reply]
Try the `files` interface then...

https://docs.directus.io/guides/interfaces.html#files

**Comment by benhaynes** [comment]
Merging this into: https://github.com/directus/directus/discussions/3001
